src/main.py

```diff
@@ -20,56 +51,69 @@
 application_user_id_mapping = {
     Application.EATERY: environ["EATERY_SLACK_USER_IDS"],
     Application.TRANSIT: environ["TRANSIT_SLACK_USER_IDS"],
-    Application.UPLIFT: environ["UPLIFT_SLACK_USER_IDS"],
+    Application.VOLUME: environ["VOLUME_SLACK_USER_IDS"],
 }
 application_slack_hook_mapping = {
     Application.COURSEGRAB: environ["SLACK_HOOK_COURSEGRAB_URL"],
     Application.EATERY: environ["SLACK_HOOK_EATERY_URL"],
     Application.TRANSIT: environ["SLACK_HOOK_TRANSIT_URL"],
-    Application.UPLIFT: environ["SLACK_HOOK_UPLIFT_URL"],
```

Comment:
would be good to keep uplift around since it's a pod next semester :)

Reply:
will make an issue since the channel is archived